src/org/usfirst/frc/team5590/robot/commands/CloseGrabber.java

```diff
@@ -0,0 +1,42 @@
+package org.usfirst.frc.team5590.robot.commands;
+
+import org.usfirst.frc.team5590.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class CloseGrabber extends Command {
+
+	public CloseGrabber() {
+		// Use requires() here to declare subsystem dependencies
+		// eg. requires(chassis);
+		requires(Robot.grabber);
+	}
+
+	// Called just before this Command runs the first time
+	protected void initialize() {
+		System.out.println("Starting CloseGrabber");
```

Comment:
Let's change this to a logger instead of a print out. It will be cleaner and follow a format. For an example, look at [this in the Autonomous Commands](https://github.com/sjcirobotics/2018_PowerUp/blob/master/src/org/usfirst/frc/team5590/robot/autonomous/basics/TurnAngle.java#L55). You'll need to create a new logger like on [Line 16 of the TurnAngle](https://github.com/sjcirobotics/2018_PowerUp/blob/master/src/org/usfirst/frc/team5590/robot/autonomous/basics/TurnAngle.java#L16), just don't forget to change the class name as well in the argument!